ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -530,4 +530,18 @@ public void shouldPrintErrorOnUnsupportedAPI() throws Exception {
         terminal.getOutputString(),
         containsString("Minimum supported client version: 1.0"));
   }
+
+  @Test
+  public void shouldListFunctions() {
+    final List<List<String>> rows = new ArrayList<>();
+    rows.add(Collections.singletonList("TIMESTAMPTOSTRING"));
+    rows.add(Collections.singletonList("EXTRACTJSONFIELD"));
+    rows.add(Collections.singletonList("CONCAT"));
+    testListOrShow("functions", TestResult.OrderedResult.build(rows), false);
+  }
+
+  @Test
+  public void shouldDescribeFunction() {
+    test("describe function timestamptostring;", EMPTY_RESULT);
```

Comment:
I assume this has to be filled in?

Reply:
We also need cases where the parameter to `DESCRIBE FUNCTION` references a non existent function.

Reply:
yeah - i still have to finish of the tests...